API/projectClass/projectClass.m

```diff
@@ -214,39 +213,31 @@
             else
                 % If the input is wrapped in a cell (so varargin is a cell of a cell)
                 % need to unwrap one layer of it, otherwise keep varargin as it is
-                if iscell(varargin{:})
+                if length(varargin) == 1 && iscell(varargin{:})
                     thisParam = varargin{:};
                 else
                     thisParam = varargin;
                 end
-                obj.parameters.addParam(thisParam);
+                obj.parameters.addParam(thisParam{:});
             end
             
         end
                 
-        function obj = removeParam(obj,varargin)
+        function obj = removeParam(obj, varargin)
```

Comment:
`removeParam` in project and parameter class should take an array of indices/names not varargin